ledger/participant-integration-api/src/test/suite/scala/platform/store/dao/events/TransactionIndexingSpec.scala

```diff
@@ -40,5 +45,125 @@ final class TransactionIndexingSpec extends AnyWordSpec with Matchers {
           .ContractWitnessesInfo(Set.empty, aDivulgence)
       )
     }
+
+    "filter out rollback nodes" in {
+      val partyStr = "Alice"
+      val party = Party.assertFromString(partyStr)
+      val bobStr = "Bob"
+      val bob = Party.assertFromString(bobStr)
+//    val offset = LedgerOffset.Absolute(LedgerString.assertFromString("offset"))
```

Comment:
Can you remove this commented line?

Reply:
6dff0ee6d9b8156e816de569e1fd4e10ab5d1a7b